modules/hibernating-cluster-resume.adoc

```diff
@@ -0,0 +1,136 @@
+csr-4vrbr  49m  kubernetes.io/kube-apiserver-client          system:node:ci-ln-812tb4k-72292-8bcj7-master-1                             24h                Approved,Issued
+csr-4wk5x  51m  kubernetes.io/kubelet-serving                system:node:ci-ln-812tb4k-72292-8bcj7-master-1                             <none>             Approved,Issued
+csr-84vb6  51m  kubernetes.io/kube-apiserver-client-kubelet  system:serviceaccount:openshift-machine-config-operator:node-bootstrapper  <none>             Approved,Issued
+----
++
+CSRs should show `Approved,Issued` in the CONDITION column.
+
+. Verify that all nodes now show as ready by running the following command:
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                      STATUS  ROLES                 AGE   VERSION
+ci-ln-812tb4k-72292-8bcj7-master-0        Ready	  control-plane,master  32m   v1.30.7
+ci-ln-812tb4k-72292-8bcj7-master-1        Ready	  control-plane,master  32m   v1.30.7
+ci-ln-812tb4k-72292-8bcj7-master-2        Ready	  control-plane,master  32m   v1.30.7
+Ci-ln-812tb4k-72292-8bcj7-worker-a-zhdvk  Ready	  worker                19m   v1.30.7
+ci-ln-812tb4k-72292-8bcj7-worker-b-9hrmv  Ready	  worker                19m   v1.30.7
+ci-ln-812tb4k-72292-8bcj7-worker-c-q8mw2  Ready	  worker                19m   v1.30.7
+----
++
+All nodes should show `Ready` in the `STATUS` column. It might take a few moments for the nodes to become ready after approving the CSRs.
+
+. Wait for cluster Operators to restart to load the new certificates.
++
+This might take 5 or 10 minutes.
+
+. Verify that all cluster Operators are in a good state by running the following command:
++
+[source,terminal]
+----
+$ oc get clusteroperators
+----
++
+.Example output
+[source,terminal]
+----
+NAME                      VERSION                             AVAILABLE  PROGRESSING  DEGRADED  SINCE   MESSAGE
+authentication            4.17.0-0.nightly-2024-12-19-020614  True       False        False     51m
+baremetal                 4.17.0-0.nightly-2024-12-19-020614  True       False        False     72m
+cloud-controller-manager  4.17.0-0.nightly-2024-12-19-020614  True       False        False     75m
+cloud-credential          4.17.0-0.nightly-2024-12-19-020614  True       False        False     77m
+cluster-api               4.17.0-0.nightly-2024-12-19-020614  True       False        False     42m
+cluster-autoscaler        4.17.0-0.nightly-2024-12-19-020614  True       False        False     72m
+config-operator           4.17.0-0.nightly-2024-12-19-020614  True       False        False     72m
+console                   4.17.0-0.nightly-2024-12-19-020614  True       False        False     55m
+...
+----
++
+All cluster Operators should show `AVAILABLE`=`True`, `PROGRESSING`=`False`, and `DEGRADED`=`False`.
+
+// .Verification
+
+// TODO: Do we want to do the basic health checking? Or is that not really necessary?
```

Comment:
@vrutkovs The test cases had a few steps of basic heath checking (step 8). Just things like logging in, creating a new project and a new app. Do you think it's worthwhile to add those here, or should we just leave it after checking that nodes/cluster operators are good to go?

Reply:
its not required for users to perform - we run these steps in CI to ensure operator status is correct